lib/user_space/uthread.c

```diff
@@ -231,8 +231,6 @@ M0_INTERNAL uint64_t m0_process(void)
 M0_INTERNAL int m0_thread_arch_adopt(struct m0_thread *thread,
 				     struct m0 *instance, bool full)
 {
-	M0_PRE(M0_IS0(thread));
-
```

Comment:
Why?

Reply:
Same reason as for `m0_thread_adopt()`, see above.

Reply:
Seems working now. Restored.